### ISSUE TITLE
predict_objects

### ISSUE THREAD

**Issue by cdancette** [OFA-Sys/OFA#189]
Hi,

Are predict_objects required for inference on the VQA task ? I don't see any mention of this in the paper.

If they are, how can we extract them for a new VQA dataset?

**Comment by yangapku** [comment]
Hi, as mentioned in the `1. Prepare the Dataset & Checkpoints` part of the `Visual Question Answering` section in OFA readme, we directly use the predicted object labels provided by [VinVL](https://github.com/pzzhang/VinVL), which bring marginal improvement (+0.1 accuracy) and are optional. For your new VQA dataset, you can directly leave the column blank or try to use the VinVL codabase to generate the labels.

**Comment by cdancette** [comment]
Would you have the checkpoints of base/large models trained without those predicted objects? 

I tried to run the inference of the vqa-finetuned base model without the objects, and the performance loss was extremely important

**Comment by yangapku** [comment]
> I tried to run the inference of the vqa-finetuned base model without the objects, and the performance was very bad.

Yes, as you mentioned, when finetuning using the object labels but making inference without using them, the result will be bad, since the input distribution is different from finetuning. You could perform your own finetuning without considering the objects simply by removing the `--add-objects` option in the finetuning shell script.
